src/putgb2.F90

```diff
@@ -2,164 +2,44 @@
 !>    @brief This subroutine read and unpack sections 6 and 7 from ah
 !>    grib2 message.
 !>    @author Stephen Gilbert @date 2002-01-11
-!>
 
 !>    This subroutine packs a single field into a grib2 message and
 !>    writes out that message to the file associated with unit lugb.
-!>    note that file/unit lugb should be opened woth a call to
-!>    subroutine baopenw before this routine is called.
+!>    Note that file/unit lugb should be opened woth a call to
+!>    subroutine baopenw() before this routine is called.
+!>
 !>    The information to be packed into the grib field is stored in a
 !>    derived type variable, gfld. gfld is of type gribfield, which is
 !>    defined in module grib_mod, so users of this routine will need to
 !>    include the line "use grib_mod" in their calling routine. Each
 !>    component of the gribfield type is described in the input argument
 !>    list section below.
 !>
-!>    PROGRAM HISTORY LOG:
-!>    - 2002-04-22 Stephen Gilbert
-!>    - 2005-02-28 Stephen Gilbert Changed dimension of array cgrib to be
-!>    a multiple of gfld%ngrdpts instead of gfld%ndpts.
-!>    - 2009-03-10Boi Vuong Initialize variable coordlist.
-!>    - 2011-06-09Boi Vuong Initialize variable gfld%list_opt.
-!>    - 2012-02-28Boi Vuong Initialize variable ilistopt.
+!>    ### Program History Log
+!>    Date | Programmer | Comments
+!>    -----|------------|---------
+!>    2002-04-22 | Stephen Gilbert | Initial.
+!>    2005-02-28 | Stephen Gilbert | Changed dim of array cgrib to be a multiple of gfld%ngrdpts instead of gfld%ndpts.
+!>    2009-03-10 | Boi Vuong | Initialize variable coordlist.
+!>    2011-06-09 | Boi Vuong | Initialize variable gfld%list_opt.
+!>    2012-02-28 | Boi Vuong | Initialize variable ilistopt.
 !>
 !>    @param[in] LUGB integer unit of the unblocked grib data file.
-!>    file must be opened with baopen or baopenw before calling this
-!>    routine.
-!>    @param[in] GFLD derived type gribfield (defined in module grib_mod)
-!>    (NOTE: See Remarks Section)
-!>    - gfld\%version GRIB edition number (currently 2)
-!>    - gfld\%discipline Message Discipline (see Code Table 0.0)
-!>    - gfld\%idsect Contains the entries in the Identification Section
-!>    (Section 1) This element is actually a pointer to an array
-!>    that holds the data.
-!>    - gfld\%idsect(1) Identification of originating Centre
-!>    (see Common Code Table C-1) 7 US National Weather Service
-!>    - gfld\%idsect(2) Identification of originating Sub-centre
-!>    - gfld\%idsect(3) GRIB Master Tables Version Number
-!>    (see Code Table 1.0) 0 Experimental; 1 Initial operational version number
```

Comment:
Just double checking, this is redundant somewhere? and thats why its getting removed?